esprima.js

```diff
@@ -2029,6 +2031,14 @@
             this.body = body;
             this.finish();
             return this;
+        },
+
+        finishYieldExpression: function (argument, delegate) {
+            this.type = Syntax.YieldExpression;
+            this.arguments = argument;
+            this.delegate = delegate;
```

Comment:
Mind sending a PR over to https://github.com/estree/estree/ to include this param in the spec?